src/_premake_init.lua

```diff
@@ -1062,6 +1062,13 @@
 		kind = "list:table",
 	}
 
+	api.register {
+		name = "props",
+		scope = "config",
+		kind = "list:table",
+		tokens = true,
+	}
```

Comment:
vstudio/_premake_init.lua seems a more appropriate place.

Reply:
Moved function definitions to `modules\vstudio\_preload.lua`